spec/lib/medical_records/user_eligibility/client_spec.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'medical_records/user_eligibility/client'
+
+describe UserEligibility::Client do
+  describe 'User eligibility operations', :vcr do
+    let(:icn) { '1000000000V000000' }
+    let(:user_id) { '10000000' }
+    let(:client) { UserEligibility::Client.new(user_id, icn) }
+
+    before do
+      Flipper.enable(:mhv_medical_records_new_eligibility_check)
```

Comment:
this should be stubbed so it's isolated to just this spec
```ruby
allow(Flipper).to_receive(:enabled).with(:mhv_medical_records_new_eligibility_check).and_return(true)
```

Reply:
Oh! I get it. I didn't understand how this was supposed to work. I understand now. Just pushed that.